### PR TITLE
Lerna to pnpm workspace migration

### PR DESCRIPTION
**Pull request by antonyagustine**
**What kind of change does this PR introduce?** (check at least one)
  - [x] Bugfix
  - [ ] Feature
  - [ ] Code style update
  - [ ] Refactor
  - [ ] Docs
  - [x] Underlying tools
  - [ ] Other, please describe:

  **Does this PR introduce a breaking change?** (check one)
  - [x] Yes
  - [ ] No

  **Version** Bump is done? (check one)
  - [ ] Yes
  - [x] No (**Reson**: It almost touches all the packages and we can bump the version on the final release with incremental patch version)

  **Other information**:
  - Optimising package installation time
  - Lerna monorepo to pnpm workspace migration

### CODE REVIEW

File: packages/contentstack-auth/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "author": "Contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "scripts": {
+    "pack": "pnpm clean && pnpm compile && oclif manifest && oclif readme",
     "build": "npm run clean && npm run compile",
```
Comment (shafeeqd959) [inline packages/contentstack-auth/package.json:9]: @antonyagustine npm pack is a built in command by npm for packaging the project, can you confirm using "pack" keyword for our scripts will effect the flow of publishing ? (npm publish internally uses npm pack command). Simple solution would be using a different keyword for it.

File: .github/workflows/sca-scan.yml [hunk 2]
```diff
@@ -7,7 +7,17 @@ jobs:
...
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          cache: 'pnpm'
+      - name: Install dependencies
+        run: pnpm install
+      # - run: npx lerna bootstrap --force-local --concurrency 1
       - name: Run Snyk to check for vulnerabilities
         uses: snyk/actions/node@master
```
Comment (shafeeqd959) [inline .github/workflows/sca-scan.yml:22]: @antonyagustine please checkout the release workflow, if it effects this change, we need to update that as well

Comment (antonyagustine) [reply .github/workflows/sca-scan.yml:22]: @shafeeqd959, @netrajpatel please ignore this PR don't spend any more time reviewing it. It is created for the POC purpose, It's still in the exploring stage.

File: packages/contentstack-auth/package.json (hunk 1 above)
Comment (antonyagustine) [reply packages/contentstack-auth/package.json:9]: @shafeeqd959 We changed the script name.

File: .github/workflows/sca-scan.yml (hunk 2 above)
Comment (antonyagustine) [reply .github/workflows/sca-scan.yml:22]: @shafeeqd959 It won't affect any release workflow.

Thanks,

**Review by netrajpatel — APPROVED** [review]

**Pull request merged**